examples/run_demo_ursula_fleet.py

```diff
@@ -0,0 +1,73 @@
+"""
+This file is part of nucypher.
+nucypher is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+nucypher is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+# WARNING This is not a mining script!
+# you will not perform any re-encryptions, and you will not get paid.
+# It might be (but might not be) useful for determining whether you have
+# the proper dependencies and configuration to run an actual mining node.
+import os
+from contextlib import suppress
+from functools import partial
+
+from twisted.internet import reactor
+
+from nucypher.characters.lawful import Ursula
+
+FLEET_POPULATION = 12
+DEMO_NODE_STARTING_PORT = 11500
+
+ursula_maker = partial(Ursula, rest_host='127.0.0.1',
+                       federated_only=True,
+                       domains=["finnegans/heartbeat/whatever"]
```

Comment:
Of course this implies also changing the Ursula test fleet's domain depending on which test is being run, which is a less generic configuration that before - Are there CI or ops follow ups associated with supporting a local fleet on another domain?

Reply:
Sounds like this is working with the new script - but we also need to maintain compatibility with the docker fleets - Have you been able to run them?  

There may be additional footwork needed to accommodate this properly (surely out of scope for "Three small bugfixes").